src/json_value_module.F90

```diff
@@ -2579,6 +2729,59 @@ subroutine wrap_json_value_add_double_vec(json, p, name, val)
     call json%add(p, to_unicode(name), val)
 
     end subroutine wrap_json_value_add_double_vec
+
+!*****************************************************************************************
+!> author: Jacob Williams
```

Comment:
If you wrote this, @jacopo-chevallard, you should probably put yourself as the author... or at least add yourself